update/firmware.php

```diff
@@ -24,6 +29,33 @@ function sendFile($path) {
     readfile($path);
 }
 
+function getLang($version_parts,$index){
+    $lang = isset($version_parts[$index])?strtolower($version_parts[$index]):'de';
+    return in_array($lang, Array('de','en','bg'))?$lang:'de';   
+}
+
+function logHeaders(){
+    file_put_contents("headers.txt", gmdate("Y-m-d H:i:s")." - ".$_SERVER['HTTP_X_ESP8266_STA_MAC']." - ".$_SERVER['HTTP_X_ESP8266_AP_MAC']." - ".$_SERVER['HTTP_X_ESP8266_FREE_SPACE']." - ".$_SERVER['HTTP_X_ESP8266_SKETCH_SIZE']." - ".$_SERVER['HTTP_X_ESP8266_CHIP_SIZE']." - ".$_SERVER['HTTP_X_ESP8266_SDK_VERSION']." - ".$_SERVER['HTTP_X_ESP8266_VERSION']."\n", FILE_APPEND | LOCK_EX);
+}
+
+function mac_filter($mac){
```

Comment:
Yes, what it does was clear after reading the code, but the name of the function was not intuitive 😄